pdpdmeetup/templates/home.html

```diff
@@ -0,0 +1,114 @@
+{% extends "base.html" %}
+
+{#{% block tabtitle %} {{ tittle }} {% endblock tabtitle %}#}
+
+{% load staticfiles %}
+
+{% block javascript %}
+    <script>
+        $(document).ready(function()
+        {
+            // Load any meetups with a status of 'upcoming' on page load
+            $.ajax({
+                type: "GET",
+                url: "{% url 'homeapp:ajax_meetups_tab' 'upcoming' %}",
+                error: function(data){
+                    $('#upcoming-meetups-tab-content').html("Ooops. I couldn't get the meetup info.");
+                    },
+                success: function(data){
+                    $('#upcoming-meetups-tab-content').html(data);
+                }
+            })
+        })
+    </script>
+
+    <script>
+        $(document).ready(function()
+        {
+            // Load any meetups with a status of 'upcoming' on page load
+            $.ajax({
+                type: "GET",
+                url: "{% url 'homeapp:ajax_meetups_tab' 'past' %}",
+                error: function(data){
+                    $('#past-meetups-tab-content').html("Ooops. I couldn't get the meetup info.");
+                    },
+                success: function(data){
+                    $('#past-meetups-tab-content').html(data);
+                }
+            })
+        })
+    </script>
+
+{% endblock %}
+
+{% block content %}
+
+	<h1>{{ group_name }}</h1>
```

Comment:
`{{ group_name }} doesn't come from anywhere. Even if it does,  it looks like it should be in `base.html`.